profiles/README.md

```diff
@@ -10,13 +10,44 @@ Specifically, we include:
 2. Processed data for each Cell Painting plate (for several "data levels")
 3. Instructions on how to reproduce the profiling pipeline
 
-## Workflow
+## Workflow 
 
-![Cytominer workflow](media/cytominer_workflow.png)
+![Cytominer workflow](media/cytominer_workflow_new.png)
+
+## Pycytominer
+[Pycytominer](https://github.com/cytomining/pycytominer) is a code base built by @gwaygenomics and @niranjchandrasekaran. 
+It allows easy processing CellProfiler data and contains all functions that were used to create the data in this repository. Below, we describe the different steps of the pipeline. Please check the pycytominer repo for more details. 
+
+Part of the pipeline, from Level 3 to Level 4b, can be found in the [profile_cells](https://github.com/broadinstitute/lincs-cell-painting/blob/master/profiles/profile_cells.py) script and the final aggregation to the consensus data is found in this [notebook](https://github.com/broadinstitute/lincs-cell-painting/blob/master/consensus/build-consensus-signatures.ipynb).  
```

Comment:
Do you think including the notebook that contains the code for sphering should also be mentioned here?

Reply:
@michaelbornholdt - can you address this?

Reply:
I have